src/main/java/nextstep/lms/service/LmsService.java

```diff
@@ -1,4 +1,27 @@
 package nextstep.lms.service;
 
+import nextstep.lms.domain.Session;
+import nextstep.lms.repository.SessionRepository;
+import nextstep.lms.repository.StudentsRepository;
+import nextstep.payments.domain.Payment;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.annotation.Resource;
+
+@Service("lmsService")
 public class LmsService {
+    @Resource(name = "sessionRepository")
+    private SessionRepository sessionRepository;
+
+    @Resource(name = "studentsRepository")
+    private StudentsRepository studentsRepository;
+
+    @Transactional
+    public void enrollStudent(Payment payment) {
+        Session session = sessionRepository.findById(payment.getSessionId());
+        if (session.enroll(payment)) {
+            studentsRepository.save(payment.getNsUserId(), payment.getSessionId());
+        }
```

Comment:
session에 수강 신청자 목록을 가지고 있어야 중복 신청을 막을 수 있지 않을까요?
sessionRepository.findById 메서드는 신청자 목록이 비어있는 상태로 초기화하는 생성자를 사용하는데요
비즈니스 로직에 적합한지 고려해 보시면 좋을 것 같아요

나아가서 enroll 메서드에서 수강 신청이 완료되었다면 DB에 저장할 수 있는 객체를 반환하면 어떨까요?
```suggestion
        final Session session = sessionRepository.findById(payment.getSessionId());
        final Student student = session.enroll(payment);
        studentsRepository.save(student);
```